docs/framework/unmanaged-api/profiling/cor-prf-gc-generation-enumeration.md

```diff
@@ -25,7 +25,7 @@ typedef enum {
     COR_PRF_GC_GEN_0 = 0,  
     COR_PRF_GC_GEN_1 = 1,  
     COR_PRF_GC_GEN_2 = 2,  
-    COR_PRF_GC_LARGE_OBJECT_HEAP = 3  
+    COR_PRF_GC_LARGE_OBJECT_HEAP = 3,
```

Comment:
I'm not seeing a new value here, just a comma.

Reply:
Sorry, I forgot to save the file before pushing apparently. The changes are added now.